### PR TITLE
hyf js week 2 homework

### PR DESCRIPTION
**Pull request by AlfiYusrina**
@segersrobbert Sorry, I did a wrong PR. But now this is the PR that contains only for week 2 homework.

### CODE REVIEW

File: Week2/homework/maartjes-work.js [hunk 1]
```diff
@@ -46,16 +46,18 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  return maartjesTasks.map(task => task.duration)
+  .map(durationInMinutes => durationInMinutes/60)
```
Comment (segersrobbertKlarrio) [inline Week2/homework/maartjes-work.js:50]: this reads like a beautiful piece of code, doesn't it? :) great job!
Small remark: indent the method calls

**Review by segersrobbertKlarrio — APPROVED** [review]

File: Week2/homework/maartjes-work.js (hunk 1 above)
Comment (AlfiYusrina) [reply Week2/homework/maartjes-work.js:50]: Ok noted. Thanks for the review. I haven't figured out why my eslint plugin on atom doesn't work properly.

File: Week2/exercises/arrayExercise.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+'use strict';
+var array = [1, 2, 3, 4, 5];
+
+//Write a JavaScript function to get the first element of an array.
+var getFirstElement = (arr) => {
+  return arr.shift();
+}
+
+//...Passing a parameter 'n' will return the first 'n' elements of the array.
+var getFirstNElements =  (array, n) => {
```
Comment (bitsagarob) [inline Week2/exercises/arrayExercise.js]: try to use a default parameter here
getFirstElement & getFirstNElements should be a single function

**Review by bitsagarob — CHANGES_REQUESTED** [review]
amazing

File: Week2/exercises/arrayExercise.js (hunk 2 above)
Comment (AlfiYusrina) [reply Week2/exercises/arrayExercise.js]: default parameter added and now in one function.

**Pull request closed without merging**